src/frontend/package.json

```diff
@@ -118,5 +120,15 @@
   },
   "resolutions": {
     "@types/react": "17.0.44"
+  },
+  "devDependencies": {
+    "@storybook/addon-actions": "^6.5.10",
+    "@storybook/addon-essentials": "^6.5.10",
+    "@storybook/addon-interactions": "^6.5.10",
+    "@storybook/addon-links": "^6.5.10",
+    "@storybook/builder-webpack5": "^6.5.10",
+    "@storybook/manager-webpack5": "^6.5.10",
+    "@storybook/react": "^6.5.10",
+    "@storybook/testing-library": "^0.0.13"
```

Comment:
In our repository, we always pin dependencies then we use renovate to manage dependency update weekly.

Reply:
Ok, I take note.